### PR TITLE
OPENINSAI: SWI-4534 - Prepare sailag.py file for upstreaming

### PR DESCRIPTION
**Pull request by ustachowiak**

### CODE REVIEW

File: ptf/sailag.py [hunk 1]
```diff
@@ -53,7 +48,8 @@ def createRemoveLagMemberTest(self):
         # verify LAG and LAG members
         attr_list = sai_thrift_get_lag_attribute(
             self.client, lag3, port_list=portlist)
-        # TODO: lag_members = attr_list["SAI_LAG_ATTR_PORT_LIST"].idlist
+        lag_members = attr_list["SAI_LAG_ATTR_PORT_LIST"].idlist
+        print(lag_members)
```
Comment (aljer) [inline ptf/sailag.py]: Did you verify it works correctly? If so, and the lag_members list is retrieved, we can also verify if the list contains all expected lag members.
\+ this print should be removed

File: ptf/sailag.py [hunk 2]
```diff
@@ -62,7 +58,8 @@ def createRemoveLagMemberTest(self):
...
```
Comment (aljer) [inline ptf/sailag.py]: same as above

File: ptf/sailag.py [hunk 3]
```diff
@@ -77,7 +74,8 @@ def createRemoveLagMemberTest(self):
...
```
Comment (aljer) [inline ptf/sailag.py]: same as above

File: ptf/sailag.py [hunk 4]
```diff
@@ -87,34 +85,25 @@ def createRemoveLagMemberTest(self):
...
```
Comment (aljer) [inline ptf/sailag.py]: same as above

File: ptf/sailag.py [hunk 5]
```diff
@@ -87,34 +85,25 @@ def createRemoveLagMemberTest(self):
...
         count = attr_list["SAI_LAG_ATTR_PORT_LIST"].count
         assert count == 0
 
-        # verify LAG and LAG members
-        attr_list = sai_thrift_get_lag_attribute(
-            self.client, lag3, port_list=portlist)
-        # TODO: lag_members = attr_list["SAI_LAG_ATTR_PORT_LIST"].idlist
-        count = attr_list["SAI_LAG_ATTR_PORT_LIST"].count
-        assert count == 0
```
Comment (aljer) [inline ptf/sailag.py]: Why is this removed?

File: ptf/sailag.py [hunk 6]
```diff
@@ -980,108 +929,102 @@ def l3LoadBalancingRemovedMembersTest(self):
...
+        print("Verify L3 load balancing with 1 member")
+        count = self.lagL3LoadBalancePacketTest([self.dev_port5])
+        print(count)
+        self.assertTrue(count[0] == self.max_itrs,
+                        "Disabled LAG member should not allow traffic")
+        self.assertTrue(
+            count[1] == 0,
+            "Enabled LAG member should take 100% traffic allow traffic")
+        self.assertTrue(count[2] == 0,
+                        "Disabled LAG member should not allow traffic")
```
Comment (aljer) [inline ptf/sailag.py]: Please unify the format

File: ptf/sailag.py (hunk 5 above)
Comment (ustachowiak) [reply ptf/sailag.py]: This is a duplicate of the fragment from lines 87-92 in the original file.

File: ptf/sailag.py [hunk 7]
```diff
@@ -0,0 +1,1350 @@
...
+                                           vlan_vid=vlan20_id,
+                                           pktlen=104)
+        tag_pkt_vlan50 = simple_udp_packet(eth_dst=dst_mac,
+                                           eth_src=src_mac,
+                                           dl_vlan_enable=True,
+                                           vlan_vid=50,
+                                           pktlen=104)
+
+        sai_thrift_set_lag_attribute(
+            self.client, self.lag1, port_vlan_id=vlan20_id)
```
Comment (pkochmanska) [inline ptf/sailag.py:595]: Shouldn't the port_vlan_id be cleaned after the test?

File: ptf/sailag.py [hunk 8]
```diff
@@ -0,0 +1,1347 @@
...
+        count = attr_list["SAI_LAG_ATTR_PORT_LIST"].count
+        assert count == 0
+
+        lag3_member24 = sai_thrift_create_lag_member(
+            self.client, lag_id=lag3, port_id=self.port24)
+        # verify LAG and LAG members
+        attr_list = sai_thrift_get_lag_attribute(
+            self.client, lag3, port_list=portlist)
+        lag_members = attr_list["SAI_LAG_ATTR_PORT_LIST"].idlist
+        # print(lag_members)
```
Comment (debasisdash-bf) [inline ptf/sailag.py]: We can remove the non-active print

File: ptf/sailag.py [hunk 9]
```diff
@@ -0,0 +1,1347 @@
...
+        print("createRemoveLagMemberTest()")
+
+        portlist = sai_thrift_object_list_t(count=100)
+        lag3 = sai_thrift_create_lag(self.client)
+
+        # verify LAG and LAG members
+        attr_list = sai_thrift_get_lag_attribute(
+            self.client, lag3, port_list=portlist)
+        lag_members = attr_list["SAI_LAG_ATTR_PORT_LIST"].idlist
+        assert len(lag_members) == 0
```
Comment (debasisdash-bf) [inline ptf/sailag.py]: Maybe we can add an assert message

File: ptf/sailag.py [hunk 10]
```diff
@@ -0,0 +1,1347 @@
...
+        count = attr_list["SAI_LAG_ATTR_PORT_LIST"].count
+        assert count == 0
```
Comment (debasisdash-bf) [inline ptf/sailag.py]: Maybe we can add an assert message

**Review by debasisdash-bf — APPROVED** [review]
Few minor comments to update.
